test/OpenTelemetry.AotCompatibility.TestApp/OpenTelemetry.AotCompatibility.TestApp.csproj

```diff
@@ -30,6 +30,7 @@
     <TrimmerRootAssembly Include="OpenTelemetry.ResourceDetectors.Process" />
     <TrimmerRootAssembly Include="OpenTelemetry.ResourceDetectors.ProcessRuntime" />
     <TrimmerRootAssembly Include="OpenTelemetry.Sampler.AWS" />
+    <TrimmerRootAssembly Include="OpenTelemetry.SemanticConventions" />
```

Comment:
Do you really want to trim `OpenTelemetry.SemanticConventions`? Fine to do but see the note on line 13, you also have to update `ci.yml` to make sure it is fired when the source changes.